dojo/models.py

```diff
@@ -1272,6 +1272,8 @@ class Finding(models.Model):
     jira_change = models.DateTimeField(editable=True, null=True)
     scanner_confidence = models.IntegerField(null=True, blank=True, default=None, editable=False, help_text="Confidence level of vulnerability which is supplied by the scannner.")
     sonarqube_issue = models.ForeignKey(Sonarqube_Issue, null=True, blank=True, help_text="SonarQube issue", on_delete=models.CASCADE)
+    vulnerabilities = models.ManyToManyField(to='Vulnerability', related_name='findings',
```

Comment:
As I haven't linked this data in yet, some tests are failing due to a missing value for this field.